<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ed_path">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>/</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrevPage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>previous page</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNextPage">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>next page</string>
    </property>
   </widget>
   <widget class="QLabel" name="labPages">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0/0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>396</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>start</string>
    </property>
    <addaction name="separator"/>
    <addaction name="linkServer"/>
   </widget>
   <widget class="QMenu" name="menudatabase">
    <property name="title">
     <string>database</string>
    </property>
    <addaction name="createDatabase"/>
    <addaction name="openDatabase"/>
    <addaction name="deleteDatabase"/>
   </widget>
   <widget class="QMenu" name="menutable">
    <property name="title">
     <string>table</string>
    </property>
    <addaction name="createTable"/>
    <addaction name="deleteTable"/>
    <addaction name="openTable"/>
    <addaction name="insertToTable"/>
    <addaction name="deleteRow"/>
    <addaction name="searchInTable"/>
    <addaction name="modifyInTable"/>
    <addaction name="open_image_from_table"/>
   </widget>
   <widget class="QMenu" name="menuspeed">
    <property name="title">
     <string>speed</string>
    </property>
    <addaction name="speedcheck_insert"/>
    <addaction name="speedcheck_image"/>
   </widget>
   <addaction name="menuStart"/>
   <addaction name="menudatabase"/>
   <addaction name="menutable"/>
   <addaction name="menuspeed"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="speedcheck_insert">
   <property name="text">
    <string>check insert</string>
   </property>
  </action>
  <action name="createDatabase">
   <property name="text">
    <string>create databse</string>
   </property>
  </action>
  <action name="deleteDatabase">
   <property name="text">
    <string>delete database</string>
   </property>
  </action>
  <action name="openDatabase">
   <property name="text">
    <string>open database</string>
   </property>
  </action>
  <action name="createTable">
   <property name="text">
    <string>create table</string>
   </property>
  </action>
  <action name="openTable">
   <property name="text">
    <string>open table</string>
   </property>
  </action>
  <action name="insertToTable">
   <property name="text">
    <string>insert into table</string>
   </property>
  </action>
  <action name="deleteTable">
   <property name="text">
    <string>delete table</string>
   </property>
  </action>
  <action name="modifyInTable">
   <property name="text">
    <string>modify from table</string>
   </property>
  </action>
  <action name="searchInTable">
   <property name="text">
    <string>search in table</string>
   </property>
  </action>
  <action name="deleteRow">
   <property name="text">
    <string>delete row from table</string>
   </property>
  </action>
  <action name="linkServer">
   <property name="text">
    <string>link to Server</string>
   </property>
  </action>
  <action name="speedcheck_image">
   <property name="text">
    <string>check image</string>
   </property>
  </action>
  <action name="open_image_from_table">
   <property name="text">
    <string>open image from table</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
